src/neptune/handler.py

```diff
@@ -726,9 +720,9 @@ def __delitem__(self, path) -> None:
 class ExtendUtils:
     @staticmethod
     def transform_to_extend_format(value):
-        """Preserve nested structure created by `Namespaces` and `dict_like` objects,
-        but replace all other values with single-element lists,
-        so work can be delegated to `extend` method."""
+        """Preserves the nested structure created by `namespaces` and `dict_like` objects,
```

Comment:
This is an internal docstring, so probably the capitalized type can stay.

```suggestion
        """Preserves the nested structure created by `Namespaces` and `dict_like` objects,
```